layouts/default/split_3x5_3/default_split_3x5_3/keymap.c

```diff
@@ -0,0 +1,14 @@
+#include QMK_KEYBOARD_H
+
+enum layers {
+    _QWERTY
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_QWERTY] = LAYOUT_split_3x5_3(
+        KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,                         KC_Y,    KC_U,    KC_I,    KC_O,   KC_P,\
+        KC_A,    KC_S,    KC_D,    KC_F,    KC_G,                         KC_H,    KC_J,    KC_K,    KC_L, KC_SCLN,\
+        KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                         KC_N,    KC_M, KC_COMM,  KC_DOT, KC_SLSH,\
+                                        KC_LGUI,   KC_BSPC,  KC_SPC,     KC_SPC,  KC_ENT,  KC_RALT \
+    )
+};
```

Comment:
```suggestion
const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
    [0] = LAYOUT_split_3x5_3(
        KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,                         KC_Y,    KC_U,    KC_I,    KC_O,   KC_P,
        KC_A,    KC_S,    KC_D,    KC_F,    KC_G,                         KC_H,    KC_J,    KC_K,    KC_L, KC_SCLN,
        KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                         KC_N,    KC_M, KC_COMM,  KC_DOT, KC_SLSH,
                                        KC_LGUI,   KC_BSPC,  KC_SPC,     KC_SPC,  KC_ENT,  KC_RALT
    )
};
```

Reply:
I can see that the layers enum has been removed and the layer is simply 0, but what's different about the layout itself? Looks identical to me.

Reply:
I've removed the backslashes from the end of the lines, they're not needed.

Reply:
I see. I'll remember that for the future. I think a bunch of keymaps still have those, since I copied the basis for this from Centromere.